docs/cookbook/absoluteFromLegacyDate.mjs

```diff
@@ -1,16 +1,8 @@
 import assert from 'assert';
 
-/**
- * Converts a Date instance into a Temporal.Absolute instance
- *
- * @param {Date} esDate This is a Date instance
- * @returns {Temporal.Absolute} Temporal.Absolute instance
- */
-function getAbsoluteFromDate(esDate) {
-  const date = Temporal.Absolute.fromEpochMilliseconds(esDate.getTime());
-  return date;
-}
+const legacyDate = new Date('1970-01-01T00:00:01Z');
```

Comment:
The function-based approach is more general and (perhaps more importantly) testable... why was it abandoned here?

Reply:
Generality isn't really a goal; this is just showing how the API can be used. The code is being tested in CI, so not sure what that's about.